pkg/reconciler/apis/apiexportendpointslice/apiexportendpointslice_controller.go

```diff
@@ -104,50 +106,52 @@ func NewController(
 		commit:                                committer.NewCommitter[*APIExportEndpointSlice, Patcher, *APIExportEndpointSliceSpec, *APIExportEndpointSliceStatus](kcpClusterClient.ApisV1alpha1().APIExportEndpointSlices()),
 	}
 
+	logger := logging.WithReconciler(klog.Background(), ControllerName)
+
 	_, _ = apiExportEndpointSliceClusterInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
-			c.enqueueAPIExportEndpointSlice(obj)
+			c.enqueueAPIExportEndpointSlice(objOrTombstone[*apisv1alpha1.APIExportEndpointSlice](obj), logger, "")
```

Comment:
tombstones only exist for `DeleteFunc`.

Reply:
Not sure I get the comment. Is it not what this function does?
Return obj or tombstone if it was deleted?

Reply:
We been using this pattern everywhere (updated,created, deletes) and it keeps it consistent (easier to read too).
So why to break this here?